arm-compute/2016-09-30/swagger/containerService.json

```diff
@@ -0,0 +1,555 @@
+              "type": "string",
+              "description": "DNS prefix to be used to create FQDN for this agent pool"
+            },
+            "fqdn": {
+              "readOnly": true,
+              "type": "string",
+              "description": "FDQN for the agent pool"
+            }
+          },
+          "required": [
+            "name",
+            "dnsPrefix"
+          ],
+          "description": "Profile for container service agent pool"
+        },
+        "ContainerServiceWindowsProfile": {
+          "properties": {
+            "adminUsername": {
+              "type": "string",
+              "description": "The administrator username to use for Windows VMs"
+            },
+            "adminPassword": {
+              "type": "string",
+              "description": "The administrator password to use for Windows VMs"
+            }
+          },
+          "required": [
+            "adminUsername",
+            "adminPassword"
+          ],
+          "description": "Profile for Windows VMs"
+        },
+        "ContainerServiceLinuxProfile": {
+          "properties": {
+            "adminUsername": {
+              "type": "string",
+              "description": "The administrator username to use for all Linux VMs"
+            },
+            "ssh": {
+              "$ref": "#/definitions/ContainerServiceSshConfiguration",
+              "description": "Specifies the ssh key configuration for Linux VMs"
+            }
+          },
+          "required": [
+            "adminUsername",
+            "ssh"
+          ],
+          "description": "Profile for Linux VMs"
+        },
+        "ContainerServiceSshConfiguration": {
+          "properties": {
+            "publicKeys": {
+              "type": "array",
+              "items": {
+                "$ref": "#/definitions/ContainerServiceSshPublicKey"
+              },
+              "description": "the list of SSH public keys used to authenticate with Linux based VMs"
+            }
+          },
+          "description": "SSH configuration for Linux based VMs running on Azure"
+        },
+        "ContainerServiceSshPublicKey": {
+          "properties": {
+            "keyData": {
+              "type": "string",
+              "description": "Certificate public key used to authenticate with VM through SSH. The certificate must be in Pem format with or without headers."
+            }
+          },
+          "required": [
+            "keyData"
+          ],
+          "description": "Contains information about SSH certificate public key data."
+        },
+        "ContainerServiceDiagnosticsProfile": {
+          "properties": {
+            "vmDiagnostics": {
+              "$ref": "#/definitions/ContainerServiceVMDiagnostics",
+              "description": "Profile for container service VM diagnostic agent"
+            }
+          }
+        },
+        "ContainerServiceVMDiagnostics": {
+          "properties": {
+            "enabled": {
```

Comment:
is this property optional? If so then by default the Diagnostics will not be enabled. Which means the default value is false. If that is indeed the case then please mention `"default": false`